examples/ota-provider-app/ota-provider-common/OTAProviderExample.cpp

```diff
@@ -91,27 +154,83 @@ void OTAProviderExample::SetOTAFilePath(const char * path)
     }
 }
 
+void OTAProviderExample::SetUserConsentDelegate(chip::ota::UserConsentDelegate * delegate)
+{
+    VerifyOrReturn(delegate != nullptr, ChipLogError(SoftwareUpdate, "User consent delegate is null"));
+    mUserConsentDelegate                      = delegate;
+    mUserConsentDelegate->userConsentCallback = OnUserConsent;
+}
+
 EmberAfStatus OTAProviderExample::HandleQueryImage(chip::app::CommandHandler * commandObj,
                                                    const chip::app::ConcreteCommandPath & commandPath,
                                                    const QueryImage::DecodableType & commandData)
 {
-    // TODO: add confiuration for returning BUSY status
+    ChipLogDetail(SoftwareUpdate, "Requestor endpoint:%hu node-id:%llu", commandPath.mEndpointId,
```

Comment:
Please use ChipLogFormatX64/ChipLogValueX64 for logging 64-bit values.

Please use PRIu16 for logging endpoint ids, instead of assuming how big a %hu is.